src/Hotel.js

```diff
@@ -23,6 +24,17 @@ class User {
     } 
     return `Username or Password was entered incorrectly`
   }
+
+  findGuestBookings(searchInput) {
+    const suspectedUser = this.allUsers.find(user => user.name.includes(searchInput))
```

Comment:
Cool, I like the way you did this. it is always cool to see things solved differently, that I would of thought of. Great work!